README.md

```diff
@@ -1,3 +1,5 @@
+# ATTENTION: This plugin has been deprecated. Please use [Koop provider hub search ogc](https://github.com/koopjs/koop-provider-hub-search-ogc) instead.
```

Comment:
Should we archive the repo as well?

Reply:
Good point! We should but I do not have admin privilege to archive the repo. I will request it from other devs.